oscarapi/views/basic.py

```diff
@@ -76,26 +71,22 @@ def get_object(self):
 
 
 class LineAttributeList(generics.ListCreateAPIView):
-    model = LineAttribute
-    queryset = LineAttribute.objects
+    queryset = LineAttribute.objects.all()
     serializer_class = serializers.LineAttributeSerializer
 
 
 class LineAttributeDetail(PutIsPatchMixin, generics.RetrieveAPIView):
```

Comment:
It is not allowed that this thing is a PutIsPatchMixin because there is no permission set on this object